yang-semver/draft-ietf-netmod-yang-semver.xml

```diff
@@ -217,6 +215,20 @@
 </ul>
 </li>
 </ul>
+<t>The _COMPAT modifier string is "sticky". Once a revision of a module 
+has a modifier in the revision label, then all descendants of that 
+revision will also have a modifier. The modifier can change from 
+_compatible to _non_compatible in a descendant revision, but the 
+modifier would never change from _non_compatible to _compatible 
+or be removed. The persistency of the _non_compatible modifier 
```

Comment:
s/persistency/persistence/

Reply:
I was more trying to indicate the quality/attribute of persistency rather than this one instance of persistence, so I'm inclined to leave it as-is (but open to be swayed here).

Reply:
I'm not sure persistency is a word.  That's the reason I suggested the change.

Reply:
I'm fine with persistence. Updated (but didn't regenerate the dates).